.github/actions/tp-e2e-tests/entrypoint.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
```

Comment:
I mentioned this on another PR, but I'm a big fan of [Unofficial Bash Strict Mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/). I've found it to be very helpful in the past.

Also, the blog post uses the shorthand version of the flags, but in scripts I intend to share I always write out the longer, more descriptive versions:

`set -o errexit -o nounset -o pipefail` (rather than `set -euo pipefail`)

Reply:
Great suggestion, this actually caught a "bug" (doesn't really change any functionality)  in our [build/functions.sh](https://github.com/apache/trafficcontrol/blob/bef00082a6b0d06ca1dc3d25996feff9acd3e362/build/functions.sh#L310) script.